### ISSUE TITLE
`bun upgrade`: attempt to unwrap error

### ISSUE THREAD

**Issue by phenomen** [oven-sh/bun#10078]
### What version of Bun is running?

1.1.2

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

1. Install Bun 1.1.2
2. `bun upgrade`

### What is the expected behavior?

Bun is upgraded to 1.1.3

### What do you see instead?

```log
C:\Users\pheno>bun upgrade
Bun v1.1.3 is out! You're on 1.1.2

uh-oh: attempt to unwrap error
bun will crash now 😭😭😭

----- bun meta -----
Bun v1.1.2 (c8d072c2) Windows x64 
UpgradeCommand: 
Elapsed: 1222ms | User: 15ms | Sys: 15ms
RSS: 77.96MB | Peak: 77.96MB | Commit: 0.14GB | Faults: 19230
----- bun meta -----

0   00007FF611E1E048
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
9   ???

Search GitHub issues https://bun.sh/issues or join in #windows channel in https://bun.sh/discord

thread 3656 panic: attempt to unwrap error
???:?:?: 0x7ff610a50c31 in ??? (bun.exe)
???:?:?: 0x7ff610a4e6d9 in ??? (bun.exe)
???:?:?: 0x7ff610a51286 in ??? (bun.exe)
???:?:?: 0x7ff610a5df67 in ??? (bun.exe)
???:?:?: 0x7ff610781603 in ??? (bun.exe)
???:?:?: 0x7ff61331e033 in ??? (bun.exe)
???:?:?: 0x7ff8e27b257c in ??? (KERNEL32.DLL)
???:?:?: 0x7ff8e346aa57 in ??? (ntdll.dll)
```

### Additional information

Installing Bun 1.1.3 with a full command (`powershell -c "irm bun.sh/install.ps1 | iex"`) works fine.

**Comment by TheAvonian** [comment]
Issue also occurs in Bun v1.1.1

**Comment by n2k3** [comment]
I'm experiencing the exact same crash when running `bun upgrade`, same `bun` and `windows` version as listed in OP.

**Comment by sphygmus** [comment]
Issue still occurs when trying to upgrade from v1.1.3 to v1.1.4.

**Comment by lekoala** [comment]
yep, same here from 1.1.1 to 1.1.4

**Comment by NehalDamania** [comment]
Same for me

**Comment by Jarred-Sumner** [comment]
This is fixed in the canary build

To install the canary build, try this:
```powershell
irm bun.sh/install.ps1 -OutFile $env:TEMP\install.ps1; & $env:TEMP\install.ps1 -Version "canary"
```

**Comment by lalo-mx** [comment]
It is crashing with 1..1.4 with Windows 10 . Latest update to 1.1.4 had the same error.
OS Version: 10.0.19045 N/A Build 19045

```
> bun upgrade --canary

uh-oh: attempt to unwrap error
bun will crash now 😭😭😭

----- bun meta -----
Bun v1.1.4 (fbe2fe0c) Windows x64
UpgradeCommand:
Elapsed: 1724ms | User: 78ms | Sys: 203ms
RSS: 122.07MB | Peak: 0.15GB | Commit: 0.18GB | Faults: 44717
----- bun meta -----

0   00007FF7511A2588
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
9   ???

Search GitHub issues https://bun.sh/issues or join in #windows channel in https://bun.sh/discord

thread 27616 panic: attempt to unwrap error
???:?:?: 0x7ff74fdb3221 in ??? (bun.exe)
???:?:?: 0x7ff74fdb0e06 in ??? (bun.exe)
???:?:?: 0x7ff74fdb3876 in ??? (bun.exe)
???:?:?: 0x7ff74fdc02fc in ??? (bun.exe)
???:?:?: 0x7ff74fae1603 in ??? (bun.exe)
???:?:?: 0x7ff7526a4c63 in ??? (bun.exe)
???:?:?: 0x7ffcfbcc7343 in ??? (KERNEL32.DLL)
???:?:?: 0x7ffcfd9426b0 in ??? (ntdll.dll)

```

Should i create a new issue?

**Comment by phenomen** [comment]
@lalo-mx it's already fixed in 1.1.5 (not released yet)
